.travis.yml

```diff
@@ -33,18 +25,22 @@ matrix:
     - php: 7.3
       env:
         - DEPS=latest
+        - CS_CHECK=true
+        - TEST_COVERAGE=true
     - php: 7.4
       env:
         - DEPS=lowest
     - php: 7.4
       env:
         - DEPS=latest
+    - php: nightly
```

Comment:
`8.0` is the correct choice here, as Travis's "nightly" target is now 8.1 development.

Reply:
OK. I will update.